source/sass_api_reference/index.html.md.erb

```diff
@@ -0,0 +1,73 @@
+---
+title: Sass API Reference
+weight: 65
+---
+
+# Sass API Reference
+
+<% format_sassdoc_data(data).each do |heading, group| %>
+
+## <%= group_heading(heading) %>
+
+<% group.each do |doc| %>
+
+### <%= doc_heading(doc) %>
+
+<%= doc.description %>
+
+<% if doc.parameter %>
+| Name | Description | Type | Default value |
+| ---- | ----------- | ---- | ------------- |
+<% parameters_table(doc.parameter).each do |param| %>
+|  <%= param.name %> | <%= param.description %> | <%= param.type %> | <%= param.default_value %> |
+<% end %>
+<% end %>
+
+#### Usage
+
+<% if doc.deprecated %>
+    <% partial(:warning_text) do %>
+        <%= markdown(doc.deprecated) %>
+    <% end %>
+<% end %>
+
+<% if doc.parameter %>
+    <% inline_parameters = "(#{inline_parameters(doc.parameter)})" %>
+<% end %>
+
+<% if doc.context.type == 'mixin' %>
+```scss
+@include <%= doc.context.name %><%= inline_parameters %><%= mixin_trailing_code(doc.context.code) %>
+```
+<% end %>
+
+<% if doc.context.type == 'function' %>
+```scss
+$result: <%= doc.context.name %><%= inline_parameters %>;
+```
+<% end %>
+
+<% if doc.context.type == 'variable' %>
+```scss
+$<%= doc.type.downcase %>: $<%= doc.context.name %>;
```

Comment:
This doesn't seem quite right, as it outputs e.g. 

```scss
$boolean: $govuk-include-default-font-face;
```

which isn't valid sass, unlike the other usage examples.

Reply:
Can you explain why it isnt valid Sass please? I agree it's a bit odd given this example but it should be valid Sass.

Reply:
I may simplify this based on your other comments regardless.

Reply:
Sorry, yes, on a second pass I can see that this is in fact valid 🤦‍♂ My brain could not make sense of it the first time, however, possibly because I didn't expect a variable to be called boolean? Or the assignment felt backwards? Not sure…

Reply:
I think I was trying to be too clever here, we should improve the Sass docs themselves instead. I'll raise issues on the main repository.